apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx

```diff
@@ -91,7 +91,7 @@ https://app.io.vtex.com/vtex.cms-builder-sf-jamstack/v1/{account}/{workspace}/bu
 <Callout type="info" emoji=" ℹ️ ">
   When an editor clicks to publish a page using the Headless CMS interface,
   the CMS calls the **Build Webhook URL**, which changes the status of that page
-  to `publishing`. The CMS, then, waits for the content to be built in the
+  to `Draft`. The CMS, then, waits for the content to be built in the
```

Comment:
I think that is `published`.
<img width="1269" alt="Captura de Tela 2023-11-17 às 11 28 28" src="https://github.com/vtex/faststore/assets/21042186/f12e3f5c-69c1-4851-8472-aa17ef3f81b9">

Reply:
Thanks, Myllena! Fixed it here